README.md

```
@@ -1,408 +1,414 @@
-   - Push the changes to the main branch:
-
-     ```bash
-     git push origin main
-     ```
-
-5. **Create GitHub Release**:
-   - Go to the GitHub repository and create a new release using the tag created in step 3.
-
-6. **Verify Release**:
-   - Verify that the release has been created successfully and that all components are working as expected.
+![Markdown Lab](docs/assets/github-banner.svg)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)